GUIDE.md

```diff
@@ -0,0 +1,159 @@
+making, it's a **topic.**
+* When the reader is consulting your document to find a specific piece of
+information, it's **reference.**
+* Each document should clearly convey *one* idea to *one* audience. All
+strategies are in service of that goal.
+
+## How We Write
+
+* Documents are written in markdown files. 
+* Those files should be written in **`lowercase-with-dashes.md`.**
+  * Underscores in filenames are allowed only when they are present in the
+    topic the document will describe (e.g., `child_process`.)
+  * Filenames should be **lowercase**.
+  * Older files may use the `.markdown` extension. These may be ported to `.md`
+    at will. **Prefer `.md` for all new documents.**
+* Documents should be word-wrapped at 80 characters.
+* Documents should not contain trailing whitespace.
+* Mechanical issues, like spelling and grammar, should be identified by tools,
+  insofar as is possible. If not caught by a tool, they should be pointed out by
+  human reviewers.
+* US spelling is preferred. "Capitalize" vs. "Capitalise", "color" vs.
+  "colour", etc.
+* Though controversial, the [Oxford comma][] is preferred for clarity's sake.
+* Generally avoid personal pronouns in reference documentation ("I", "you",
+  "we".)
+  * Pronouns are acceptable in more colloquial documentation, like guides.
+* Place end-of-sentence punctuation inside wrapping elements – periods go
+  inside parentheses and quotes, not after.
+* Documents must start with a level-one heading. An example document will be
+  linked here eventually.
+* Prefer affixing links to inlining links – prefer `[a link][]` to
+  `[a link](google dot com)`.
+* When documenting APIs, note the version the API was introduced at
+  the end of the section. If an API has been deprecated, also note the first
+  version that the API appeared deprecated in.
+
+### Tone
+
+The tone of your writing should change in service of the goal of the document.
+For example, *guides* may be warm and whimsical in order to make the reader
+comfortable and pique their interest. *Reference material* should adopt a
+terser, more clinical tone, in order to more efficiently convey information to
+the reader. *Topics* are halfway between the two – a more neutral tone may be
+adopted, but it's okay to mix some humanity in as well!
+
+### Introducing Changes
+
+To introduce changes, open a PR to this repository. One "LGTM" comment from
+another team member is sufficient to merge a change. When describing your
+change, make sure to note why you are making the change in the description. If
+you are writing a new doc, please include a one-to-three sentence paragraph
+explaining the idea you are attempting to convey and the audience you are
+targeting. For more information on this, see [CONTRIBUTING.md][].
+
+[as fun]: https://twitter.com/izs/status/187639633641865216
+[Oxford comma]: https://en.wikipedia.org/wiki/Serial_comma
+[CONTRIBUTING.md]: CONTRIBUTING.md
```

Comment:
I am working on the CONTRIBUTING.md guide in a separate PR so that we can incrementally approve changes.